src/CoreBundle/Resources/views/snippets-cms/NavigationTree/standard.html.twig

```diff
@@ -0,0 +1,454 @@
+{% endif %}
+        <div class="row">
+            <div class="col-12 col-md-6 col-xl-8">
+                <div id="navigationTreeContainer{% if noAssignDialog == false %}-checkboxes{% endif %}" class="navigationTreeContainer"></div>
+            </div>
+            <div class="col-12 col-md-6 col-xl-4 justify-content-end mt-4 mt-md-0">
+                <div class="treelegend">
+                    <h5>
+                        <span class="nodeIndicatorIcon"></span>
+                        <span>{{ 'chameleon_system_core.cms_module_page_tree.legend_header' | trans }}</span>
+                    </h5>
+
+                    {% if noAssignDialog == false %}
+                        <div class="activeConnectedNode primaryConnectedNode">
+                            <span class="nodeIndicatorIcon"></span>
+                            <span>{{ 'chameleon_system_core.cms_module_page_tree.legend_active_main_node_of_page' | trans }}</span>
+                        </div>
+
+
+                        <div class="activeConnectedNode">
+                            <span class="nodeIndicatorIcon"></span>
+                            <span>{{ 'chameleon_system_core.cms_module_page_tree.legend_active_further_node_of_page' | trans }}</span>
+                        </div>
+                    {% endif %}
+
+                    <div class="text">
+                        <span class="nodeIndicatorIcon"></span>
+                        <span>{{ 'chameleon_system_core.cms_module_page_tree.legend_node_has_no_page' | trans }}</span>
+                    </div>
+
+                    <div class="otherConnectedNode">
+                        <span class="nodeIndicatorIcon"></span>
+                        <span>{{ 'chameleon_system_core.cms_module_page_tree.legend_has_connected_pages' | trans }}</span>
+                    </div>
+
+                    <div class="restrictedPage">
+                        <span class="nodeIndicatorIcon iconRestricted"><i class="fas fa-lock"></i></span>
+                        <span>{{ 'chameleon_system_core.cms_module_page_tree.legend_connected_to_protected_page' | trans }}</span>
+                    </div>
+
+                    <div class="node-hidden">
+                        <span class="nodeIndicatorIcon iconHidden"><i class="far fa-eye-slash"></i></span>
+                        <span>{{ 'chameleon_system_core.cms_module_page_tree.legend_hidden' | trans }}</span>
+                    </div>
+
+                    <div class="legendLine">
+                        <span class="nodeIndicatorIcon iconExternalLink"><i class="fas fa-external-link-alt"></i></span>
+                        <span>{{ 'chameleon_system_core.cms_module_page_tree.legend_external_link' | trans }}</span>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+
+{% if isInIframe == false %}
+    </div>
+</div>
+{% endif %}
+
+<script>
+    var currentNodeId = null;
+
+    $("#navigationTreeContainer")
```

Comment:
The javascripts should move to a JS include. The JS files should load the necessary variables from PHP using data attributes of the tree div container. And there is a general way to get translations in javascripts like this: 
`CHAMELEON.CORE.i18n.Translate('chameleon_system_media_manager.tree_context_menu.edit_properties')`

Reply:
ok, all js were moved to navigationTree.js
Translations that are used in js were moved to admin-js.de/en.xliff
Some translations in admin.de/en.xliff are now deprecated and labeled accordingly.